resources/monitoring/charts/alert-rules/templates/kyma-rules.yaml

```diff
@@ -33,11 +33,31 @@ groups:
 - name: pvc-90-percent-full-rule
   rules:
   - alert: PVC90PercentFull
-    expr: kubelet_volume_stats_used_bytes{namespace="kube-system",exported_namespace=~"kyma-.*|kube-.*|istio-.*|heptio-ark" } / kubelet_volume_stats_capacity_bytes{namespace="kube-system",exported_namespace=~"kyma-.*|kube-.*|istio-.*|heptio-ark"} * 100 > 90
+    expr: kubelet_volume_stats_used_bytes{namespace="kube-system",exported_namespace=~"kyma-.*|kube-.*|istio-.*|heptio-ark|natss" } / kubelet_volume_stats_capacity_bytes{namespace="kube-system",exported_namespace=~"kyma-.*|kube-.*|istio-.*|heptio-ark|natss"} * 100 > 90
     for: 10m
     labels:
       severity: critical
     annotations:
       description:  "PVC {{`{{$labels.exported_namespace}}`}}/{{`{{$labels.persistentvolumeclaim}}`}} is using {{`{{$value}}`}} % of the available volume"
       summary: "PVC {{`{{$labels.exported_namespace}}`}}/{{`{{$labels.persistentvolumeclaim}}`}} is using {{`{{$value}}`}} % of the available volume"
+- name: cpu-90-percent-usage-rule-natss
```

Comment:
can we use the existing rule and modify it for natss ? Because the rules look duplicate.

Reply:
Done.